core/templates/filters/string-utility-filters/normalize-whitespace-punctuation-and-case.pipe.ts

```diff
@@ -28,8 +28,8 @@ import { Injectable, Pipe, PipeTransform } from '@angular/core';
 @Pipe({name: 'normalizeWhitespacePunctuationAndCase'})
 export class NormalizeWhitespacePunctuationAndCasePipe
 implements PipeTransform {
-  transform(input: any): any {
-    if (typeof input === 'string' || input instanceof String) {
+  transform(input: string): string {
+    if (typeof input === 'string') {
```

Comment:
Do we still need this check? It should be checked by the TypeScript right?

Reply:
I think we need this if we get some invalid input in runtime like null, undefined than the function should just return that invalid value.